pyproject.toml

```
@@ -17,6 +17,12 @@ test_gis = [
     "pytest",
     "momepy",
 ]
+rl_example = [
+    "stable-baselines3",
+    "seaborn",
+    'mesa",
+    "tensorboard"
+]
```

This is purely for the tests in CI. I think that should be:

```suggestion
rl_test = [
    "pytest",
    "stable-baselines3",
    "tensorboard"
]
```
Mesa is separately installed and seaborn is probably not used in the model run itself.

Which file should we run if we want to properly test this code in CI? We will be stepping the model a few times, but do we also need to test/run other files?